src/components/common/messaging/MessageBox.tsx

```diff
@@ -185,12 +188,20 @@ export default observer(({ channel }: Props) => {
             }
         }
 
+        const textBox = document.getElementById("message");
+        textBox?.addEventListener("compositionstart", () => {
+            setComposed(false);
+        });
+        textBox?.addEventListener("compositionend", () => {
+            setComposed(true);
+        });
+
         return internalSubscribe(
             "MessageBox",
             "append",
             append as (...args: unknown[]) => void,
         );
-    }, [draft, setMessage]);
+    }, [draft, setMessage, composed]);
```

Comment:
This dependency change is also unneeded since the added code doesn't depend on `composed`, although it does depend on `setComposed`.